packages/engine/CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+[comment]: <> (This CONTRIBUTING guide was heavily inspired by the great one provided by the lovely folks at Atom [https://github.com/atom/atom/blob/master/CONTRIBUTING.md])
+
+# Contributing to hEngine
+
+Thank you for your interest in adding to the hEngine!
+
+We've provided the following guidelines to help you with contributions, in addition to the ones described below, we've
+also established a set of [community guidelines](https://hash.ai/legal/community) to enable as many people as possible
+to contribute to and benefit from HASH. Please follow these when interacting with this repo.
+
+#### Table Of Contents
+
+- [I have a question, where do I go?](#i-have-a-question--where-do-i-go-)
+
+- [Getting started, what do I need to know?](#getting-started--what-do-i-need-to-know-)
+    * [Design Decisions](#design-decisions)
+
+- [How Can I Contribute?](#how-can-i-contribute-)
+    * [Reporting Bugs](#reporting-bugs)
+        + [Before Submitting A Bug Report](#before-submitting-a-bug-report)
+        + [How Do I Submit A (Good) Bug Report?](#how-do-i-submit-a--good--bug-report-)
+    * [Suggesting Enhancements](#suggesting-enhancements)
+        + [Before Submitting An Enhancement Suggestion](#before-submitting-an-enhancement-suggestion)
+        + [How Do I Submit A (Good) Enhancement Suggestion?](#how-do-i-submit-a--good--enhancement-suggestion-)
+    * [Your First Code Contribution](#your-first-code-contribution)
+        + [Local development](#local-development)
+    * [Pull Requests](#pull-requests)
+
+- [Styleguides](#styleguides)
+    * [Rust Styleguide](#rust-styleguide)
+    * [JavaScript Styleguide](#javascript-styleguide)
+    * [Python Styleguide](#python-styleguide)
+    * [Documentation Styleguide](#documentation-styleguide)
+        + [Example](#example)
+
+- [Additional Notes](#additional-notes)
+    * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+## I have a question, where do I go?
+
+> **Note:** Please don't use GitHub issues for help, we have a friendly community on... TODO links to Discord, etc.
+
+* TODO
+
+## Getting started, what do I need to know?
+
+### Design Decisions
+
+> TODO
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+> TODO
+
+#### Before Submitting A Bug Report
+
+> TODO
+
+#### How Do I Submit A (Good) Bug Report?
+
+> TODO
+
+### Suggesting Enhancements
+
+> TODO
```

Comment:
We're going to have the ability to submit suggestions via a 'wishlist' section of the new site. https://app.asana.com/0/1200211978612931/1201468117002661 In the meantime I think GitHub Issues would be an appropriate place to direct people, as well - and more structured/preferable than Discord.

Reply:
We also have the product feedback section of the forum, but this will be merged into _general_ when the 'wishlist' site is up. Perhaps it is, in fact, the best place to point people for now: https://community.hash.ai/c/product-feedback/2